modules/plot/main/coffee/Utils.coffee

```diff
@@ -1,3 +1,19 @@
+doCollisionDetection = (nodes) ->
+  distance = 1
+  nodes.forEach (node, i) ->
+    previousNodes = nodes.slice 0, (i - distance + 1)
+    doesOverlap = previousNodes.find (previousNode, j) ->
+      prevBox = previousNode.getBoundingClientRect()
+      currBox = node.getBoundingClientRect()
+      doesCollide = currBox.left < prevBox.right
+      currentDistance = i - j
+      if doesCollide and currentDistance > distance
+        distance = currentDistance
+      doesCollide
+  nodes.forEach (node, i) ->
+    if i % (distance + 1)
+      hx.select node
+        .text ''
```

Comment:
I would prefer to leave it like this - nthTickVisible doesn't remove the label, so I don't think we should do so here either.